app/manifest.json

```diff
@@ -60,6 +68,7 @@
     "clipboardWrite",
     "http://localhost:8545/",
     "https://*.infura.io/",
+    "*://connect.trezor.io/5/*",
```

Comment:
I'm gonna try to take this permission of and see if stills works.
In theory if we're allowed to inject web3 on any website, we should be allowed to inject code on connect.trezor.io

Reply:
YAY! we don't need it!